guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,301 @@
+{
```

Comment:
<p>Data visualization can provide useful information for timeseries modeling. For example, it can show if any data is missing, which could impact a model's efficacy. </p><p>It could also reveal the seasonality of the data to some extent. For instance, it could make the time of peaks and troughs more obvious, which can help in determining the lag/sequence length for data initialization, a very significant parameter for modeling timeseries data. </p><p>This visualization is done by data.show_batch().  All time series data must have a datetime column that data.show_batch() will use to plot time series data, with the time stamp in the x - axis and the variable in y axis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>